.github/workflows/UpdatePropertiesTableInMSDocs.yaml

```diff
@@ -0,0 +1,78 @@
+# This workflow is used to update the configuration properties files MS docs repo: https://github.com/MicrosoftDocs/azure-dev-docs-pr/tree/main/articles/java/spring-framework/includes/spring-cloud-azure
```

Comment:
1. `This workflow is used to` -> `This workflow will create PR to`
2. `files MS docs repo` -> `files in MS docs repo`
3. 
```
# For example, this PR is created by this workflow: https://github.com/MicrosoftDocs/azure-dev-docs-pr/pull/3035
```